trail/query/urls.py

```diff
@@ -1,11 +1,30 @@
-from query.views import print_results, index
-from django.urls import path
-
+from query.views import print_results, index, MetadataViewSet, get_metadata
```

Comment:
`MetadataViewSet` doesn't exist in `query.views` but something called `MetaDataQuery` does. CamelCase is usually best used to separate words, so `MetadataQuery` would be the capitalization.